apps/election-manager/src/utils/filenames.test.ts

```diff
@@ -64,3 +67,89 @@ test('generates ballot export package name with zero padded time pieces', () =>
     'king-county_general-election_testHash12__2019-03-01_01-09-02.zip'
   )
 })
+
+describe('parseCVRFileInfoFromFilename', () => {
+  test('parses a basic name not in test mode properly', () => {
+    const name = 'machine_5__1_ballots__2020-12-08_10-42-02.jsonl'
+    const results = parseCVRFileInfoFromFilename(name)
+    expect(results)
+    expect(results?.machineId).toBe('5')
+    expect(results?.numberOfBallots).toBe(1)
+    expect(results?.isTestModeResults).toBe(false)
+    expect(results?.timestamp).toStrictEqual(new Date(2020, 11, 8, 10, 42, 2))
```

Comment:
I usually find it's better to do these assertions in one go so you get more context around a failure.
```suggestion
    expect(results).toEqual({
      isTestModeResults: false,
      machineId: '5',
      numberOfBallots: 1,
      timestamp: new Date(2020, 11, 8, 10, 42, 2),
    })
```